.github/workflows/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Was this added by the "act" tool? If yes, remove this file, if not, what was the reason?